pipenv/core.py

```diff
@@ -1503,7 +1503,7 @@ def pip_install(
         "PIP_DESTINATION_DIR": vistir.misc.fs_str(
             cache_dir.joinpath("pkgs").as_posix()
         ),
-        "PIP_EXISTS_ACTION": vistir.misc.fs_str("w"),
+        "PIP_EXISTS_ACTION": vistir.misc.fs_str(PIP_EXISTS_ACTION or "w"),
```

Comment:
Is this a good idea? The line in `environments.py` sets `w` when the variable is missing, so maybe w should respect the user explicitly setting it to `` (empty string).

Reply:
That would just mean using the pip defaults which seems fine to me, I don't think it matters much -- probably better to be consistent and do less hidden assumptions

Reply:
@uranusjr may be right, we should leave it to PIP, in case it gets overriden by empty string.
However, explicit is better than implicit.
Your choice.

Reply:
Honestly I doubt whether anyone would care about this extreme edge case, so in the end it’s only a theoretical preference.

Reply:
I'll be happy to merge this either way, planning to do so once CI passes